### PR TITLE
Add missing key count metric

### PR DESCRIPTION
**Pull request by terryyylim**
**What this PR does / why we need it**:
[missingKeyCount](https://github.com/feast-dev/feast/blob/master/serving/src/main/java/feast/serving/util/Metrics.java#L41 ) metric used to be available in Prometheus but was removed during refactor to OnlineServingService and RedisOnlineRetriever, from [RedisServingService](https://github.com/feast-dev/feast/blob/cc884b4480c50555d8a9f28e0629d138d0d9bd36/serving/src/main/java/feast/serving/service/RedisServingService.java#L220).

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: serving/src/main/java/feast/serving/util/Metrics.java [hunk 1]
```diff
@@ -40,20 +40,12 @@
 
   public static final Counter missingKeyCount =
```
Comment (zhilingc) [inline serving/src/main/java/feast/serving/util/Metrics.java:41]: Maybe just rename this to match the metric name?

Comment (terryyylim) [reply serving/src/main/java/feast/serving/util/Metrics.java]: Renamed.

**Pull request merged**